README.md

```diff
@@ -117,6 +117,12 @@ customize it.
 
 ## Development and testing
 
+You may use the flag `--help` to view all supported features.
+
+### Debugging
+
+The flag `--debug` enables installation of OpenSK in debug mode. You may use the GUI application J-Link RTT viewer to see the debugging messages.
```

Comment:
One nit and one comment.

nit: We usually break longer lines.

Since you mention the J-Link RTT viewer, may I ask you to talk more about what that is and how to get it? Or is that already somewhere in our documentation?

Reply:
Also, RTT Viewer is only enabled on the devkit. The dongles are using UART pins instead.

Reply:
Sure, I can extend it :). Does it mean the debug is not enabled for dongles?

Reply:
Debug can be enabled on dongles too with `--debug` but it will happen over another channel than RTT (i.e. J-Link RTT Viewer won't work on dongles by default)

Reply:
I think I understand it now. For dongles some external hardware (e.g. J-Link probe) is needed for debugging, which is going through UART pins?

Reply:
External hardware is needed. The JTAG probe such as a J-Link one will have to be wired on the debug port and the console will require an external USB to UART device to be wired to the dedicated pins.

Reply:
Yes ^^ Understood